telemetry_package/main.py

```diff
@@ -1,28 +1,74 @@
-import sys
+import json
 import rclpy
-from rclpy.node import Node
+import socket
+import os
+import select
 
+from rcl_interfaces.msg import Log
 from common_package_py.common_node import CommonNode
 
+DEFAULT_UNIX_SOCKET_PATH = "/tmp/thi_drone"
 
 class TelemetryNode(CommonNode):
```

Comment:
@Philipp66904 said, this wouldn't be necessary to him, as the node isn't located on the drone

Reply:
In fact, I actually don't want @maf0115 to use it, as this will result in problems with the heartbeat (the Quality of Service is set wrong for the telemetry node and Mission Control would produce a lot of error messages as it doesn't expect any heartbeats from you).